### PR TITLE
fix: update to newest IPLD libraries

### PR DESCRIPTION
**Pull request by vmx**
ipld-dag-pb got a new release with breaking changes, update to that release.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,11 +60,11 @@
     "err-code": "^1.1.2",
     "hamt-sharding": "~0.0.2",
     "ipfs-unixfs": "~0.1.16",
-    "ipld-dag-pb": "~0.17.2",
+    "ipld-dag-pb": "^0.18.0",
     "multicodec": "~0.5.1",
     "multihashing-async": "~0.7.0",
-    "superstruct": "~0.6.1",
-    "rabin-wasm": "~0.0.4"
+    "rabin-wasm": "0.0.8",
```
Comment (achingbrain) [inline package.json]: Any reason why this is not prefixed with `~`?

**Comment by achingbrain** [comment]
Looks good, though there are some linting errors to take care of before it can be merged.

File: package.json (hunk 1 above)
Comment (vmx) [reply package.json]: The reason was an `npm install rabin-wasm@latest`. So it should really be `~0.0.8`.

**Comment by vmx** [comment]
I'll fix those.

**Review by achingbrain — APPROVED** [review]

**Pull request merged**